02-overview-surveys.Rmd

```diff
@@ -119,7 +119,7 @@ As part of our survey on the average number of pets in a household, researchers
 >
 > </ul>
 
-This question may have validity issues as it only provides the options of "dogs" and "cats" to respondents, and interpretation of the data could be incorrect. For example, if we had 100 respondents who answered the question and 50 selected dogs, then the results of this question cannot be `50% of the population prefers to have a dog as a pet` as only two response options were provided. If a respondent taking our survey prefers turtles, they could either be forced to choose a response between these two (i.e., interpret the question as "between dogs and cats, which do you prefer?" and result in *measurement error*), or they may not answer the question (which results in *item nonresponse error*). Based on this, the interpretation of this question should be `When given a choice between dogs and cats, 50% of respondents preferred to have a dog as a pet`.
+This question may have validity issues as it only provides the options of "dogs" and "cats" to respondents, and interpretation of the data could be incorrect. For example, if we had 100 respondents who answered the question and 50 selected dogs, then the results of this question cannot be `50% of the population prefers to have a dog as a pet` as only two response options were provided. If a respondent taking our survey prefers turtles, they could either be forced to choose a response between these two (i.e., interpret the question as "between dogs and cats, which do you prefer?" and result in *measurement error*), or they may not answer the question (which results in *item nonresponse error*). Based on this, the interpretation of this question should be `When given a choice between dogs and cats, 50% of respondents preferred to have a dog as a pet`. 
```

Comment:
Using the `` anywhere is hard to identify in the book.  It really stood out here, thinking this should be changed to "" instead.  In other places we use `` for variable names etc should we bold them to make them stand out more?  Thoughts?